website/docs/development/testing.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: Testing
+---
+
+## Testing
+
+As of v1.19.2, GitProxy uses [Mocha](https://mochajs.org/) (`ts-mocha`) as the test runner, and [Chai](https://www.chaijs.com/) for unit test assertions. End-to-end (E2E) tests are written in [Cypress](https://docs.cypress.io), and some fuzz testing is done with [`fast-check`](https://fast-check.dev/).
```

Comment:
Are the Cypress tests E2E? They're not handling the proxy  but the rather just the admin UI and hence only part of application.

```suggestion
As of v1.19.2, GitProxy uses [Mocha](https://mochajs.org/) (`ts-mocha`) as the test runner, and [Chai](https://www.chaijs.com/) for unit test assertions. User interface tests are written in [Cypress](https://docs.cypress.io), and some fuzz testing is done with [`fast-check`](https://fast-check.dev/).
```

Reply:
Good point! I thought of them as E2E since they handle the _entire_ flow from UI to API and back again. But you're right that the proxy code never gets tested here, and there is currently no way to test the proxy through the UI...